mesonbuild/modules/gnome.py

```diff
@@ -224,12 +224,13 @@ class MkEnums(_MkEnumsCommon):
 ]
 
 def annotations_validator(annotations: T.List[T.Union[str, T.List[str]]]) -> T.Optional[str]:
-
     """Validate gdbus-codegen annotations argument"""
 
     badlist = 'must be made up of 3 strings for ELEMENT, KEY, and VALUE'
 
-    if all(isinstance(annot, str) for annot in annotations):
+    if not annotations:
+        return None
+    elif all(isinstance(annot, str) for annot in annotations):
```

Comment:
This line doesn't need to be modified into an elif (instead of `-1 +3` it could just be `+2`) because it should always run, except when we returned early.

Reply:
I'm really not suer what the concern here is? I'm adding another condition to a series of if/elif conditions, and it happens to be the first one so the second becomes an elif, this seems straightforward to me.

Reply:
There's no actual concern, just a random comment. It doesn't make a difference, but stylistically I would have done it different.